cmake/projects/YAML_CPP/hunter.cmake

```diff
@@ -0,0 +1,21 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  YAML_CPP
```

Comment:
Yes. I replace it in the hunter-package. Can I do it directly or you want to review it?

And I think `yaml-cpp` will be at good plae in `default.cmake`

Anyway, do you have "standard" for package names?